### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Ju999** [MinekCZ/PalermoTown#3]
When you setup the arena and restart the Server it closes and says `"Can't place chest in unloaded Worlds"`.

I played with the source code a bit and if you add a 
`sleep(1)` at line 627 in `Arena.php`, it does this:
`ParseError: "syntax error, unexpected variable "$chests"" (EXCEPTION) in "plugins/PalermoTown_dev-23/src/MinekCz/PalermoTown/Arena" at line 628` idk if it helps. If you need help with anything else, you can message me on discord (Jui999#4488)!

I would love to see this Bug being fixed, so this amazing Plugin is working!

**Comment by Tuvqlu** [comment]
Yeah try again, finally worked for me yesterday. Looks like you might have set up your chests wrong
